src/index.js

```diff
@@ -26,10 +27,10 @@ function toVal(mix) {
 	return str;
 }
 
-export function clsx() {
-	var i=0, tmp, x, str='';
-	while (i < arguments.length) {
-		if (tmp = arguments[i++]) {
+export default function () {
```

Comment:
breaking change, need named

```suggestion
export function clsx() {
```